ir/ir.def

```diff
@@ -226,8 +230,8 @@ class Entry : IAnnotated {
 
 /// List of predefined entries. Part of table properties
 class EntriesList : PropertyValue {
-    Vector<Entry> entries;
-    size_t size() const { return entries->size(); }
+    inline Vector<Entry> entries;
```

Comment:
Do we have any non-inline Vectors or IndexedVectors left? (Hard to tell from the diff.)
If all vectors are inline, could this be enforced in the IR design somehow? This would simplify the programmer's task - you would not have to look up the def file to know whether something is inline or not as often.